moped-editor/src/views/projects/projectView/ProjectComponents.js

```diff
@@ -185,124 +186,107 @@ const ProjectComponents = () => {
-                                  </TableCell>
-                                  <TableCell>
-                                    {[
-                                      ...new Set(
-                                        component.moped_proj_components_subcomponents.map(
-                                          mpcs =>
-                                            mpcs.moped_subcomponent
-                                              .subcomponent_name
-                                        )
-                                      ),
-                                    ]
-                                      .sort()
-                                      .join(", ")}
-                                  </TableCell>
-                                  <TableCell align={"center"}>
-                                    <DoubleArrowIcon
-                                      onClick={handleComponentDetailsClick}
-                                    />
-                                  </TableCell>
-                                </TableRow>
-                              );
-                            }
-                          )}
-                        </TableBody>
-                      </Table>
-                    </TableContainer>
-                  </ClickAwayListener>
-                </Paper>
-              )}
-              {!componentsAvailable && (
-                <Alert severity="info">
-                  There aren't any components for this project.
-                </Alert>
-              )}
-              <Button
-                className={classes.componentButtonAddNew}
-                onClick={handleAddNewComponentClick}
-                variant="outlined"
-                color="default"
-                size={"large"}
-                startIcon={<Icon>add</Icon>}
-              >
-                Add new component
-              </Button>
-            </Grid>
+      {!componentEditMode && (
+        <ErrorBoundary
+          FallbackComponent={({ error, resetErrorBoundary }) => (
+            <ProjectSummaryMapFallback
+              error={error}
+              resetErrorBoundary={resetErrorBoundary}
+              projectId={projectId}
+              setIsEditing={null}
+              refetchProjectDetails={refetch}
+              mapData={projectFeatureCollection}
+            />
+          )}
+          onReset={() => setMapError(false)}
+          resetKeys={[mapError]}
+        >
+          <ProjectComponentsMapView
+            projectFeatureCollection={projectFeatureCollection}
+            setIsEditing={false}
+            noPadding
+          >
+            <Button
+              className={classes.componentButtonAddNew}
+              onClick={handleAddNewComponentClick}
+              variant="outlined"
+              color="default"
+              size={"large"}
+              startIcon={<Icon>add</Icon>}
+              fullWidth
+            >
+              Add new component
+            </Button>
```

Comment:
Basically we get rid of the Table family of components for the List family of components.